google/protobuf/duration.proto

```diff
@@ -0,0 +1,104 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Never mind, we have a fix in the grpc tree. I'll modify this PR soon and remove local copy of google/protobuf.